File: OCH_Tao/ABC/B/019.py
S = list(input())
x = S[0]
cnt = 0
l = []
for i in S:
  if i != x:
    l.append(x+str(cnt))
    x = i
    cnt = 1
  else:
    cnt += 1
else:
  l.append(x+str(cnt))
print("".join(l))

File: OCH_Tao/ABC/B/019.js
const S = [...(require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0]];
let x = S[0];
let cnt = 0;
let l = [];
for (let i = 0; i < S.length; i++) {
  if (S[i] !== x) {
    l.push(x + String(cnt));
    x = S[i];
    cnt = 1;
  } else {
    cnt++;
  }
}
l.push(x + String(cnt));
console.log(l.join(""));